R/retrieve.R

```diff
@@ -28,10 +28,12 @@ retrieve <- function(packages) {
   }
   renv_scope_options(HTTPUserAgent = agent)
 
+  writef("Downloading %s package(s) and their dependencies", length(packages))
```

Comment:
```suggestion
  writef("Downloading %s and their dependencies.", nplural("package", length(packages)))
```

Reply:
That doesn't quite work because in the one package case it should really be "Downloading 1 package and its dependencies".  We could use `nggettext()` here or create some helper:

```R
writef(
 ngettext(
   length(packages),
   "Downloading %s package and its dependencies",
   "Downloading %s packages and their dependenices",
 ),
 length(packages)
)
```

But that's going to be a lot of work.

Reply:
Good point. Let's leave it alone.